site/reference/cli/fn/sink/README.md

```diff
@@ -30,7 +30,9 @@ kpt fn sink [DIR] [flags]
 
 DIR:
   Path to a local directory to write resources to. Defaults to the current
-  working directory. Directory must exist.
+  working directory.
+  1. If the directory doesn't already exist, it will be created.
+  2. If the directory already exists, the existing files in it are not deleted.
```

Comment:
I thought the AI we discussed was to error out instead?

Reply:
Updated logic to error out if directory already exists.